x-pack/plugins/canvas/server/collectors/workpad_collector.ts

```diff
@@ -44,6 +44,10 @@ interface WorkpadTelemetry {
       max: number;
     };
   };
+  templates?: {
+    total: number;
+    by_name: Record<string, number>;
```

Comment:
Someone will want to double-check me that this is a way we can query and aggregate these numbers in Kibana...

Reply:
I'm not sure this will work because we would be creating an index with unknown keys (like, we won't know all of the names of templates), kind of like how we don't map function use by this either.  But maybe should check with the telemetry team and see if there is a proper way to do something like this.